docs/source/concepts/glossary.rst

```diff
@@ -393,25 +451,75 @@ As a user, you don't need to interact with Daml-LF directly. But internally, it'
 - sending and receiving values via the Ledger API (using a protocol such as gRPC)
 - generating code in other languages for interacting with Daml models (often called “codegen”)
 
-General concepts
-****************
+Composability
+=============
 
-Ledger, Daml ledger
-===================
+Composability is the ability of a participant to extend an existing system with new Daml applications or new topologies unilaterally without requiring cooperation from anyone except the 
+directly involved participants who wish to be part of the new application functionality.
 
-**Ledger** can refer to a lot of things, but a ledger is essentially the underlying storage mechanism for a running Daml applications: it's where the contracts live. A **Daml ledger** is a ledger that you can store Daml contracts on, because it implements the `ledger API <#ledger-api>`__.
 
-Daml ledgers provide various guarantees about what you can expect from it, all laid out in the :doc:`/concepts/ledger-model/index` page.
 
-When you're developing, you'll use `Sandbox <#sandbox>`__ as your ledger.
 
-.. _trust-domain:
 
-Trust domain
-============
+Canton Concepts
+***************
+
+.. Language for the participant node was taken, and edited, from the Canton key terms page
+Participant Node
```

Comment:
Are you sure you want all of these concepts under "Canton"? Some of them are generic concepts not directly tied to Canton.

Reply:
agree, I moved the participant node section out of the Canton section and into the 'building applications' section